<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HumanEditDialog</class>
 <widget class="QDialog" name="HumanEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="7" column="0">
          <widget class="QCheckBox" name="checkBoxLife">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Живой?</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBoxSex"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Дата рожения:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Имя: </string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Род занятий:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditSurName"/>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEditBirthPlace">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSearchPlace">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Отчество:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Пол:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEditOccupation"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Место рождения:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEditBirthdate">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditName"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditMarName"/>
         </item>
         <item row="17" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Заметки:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="horizontalLayoutDeathDate">
           <item>
            <widget class="QLabel" name="labelDeathDate">
             <property name="text">
              <string>Дата смерти:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDeathDate"/>
           </item>
           <item>
            <widget class="QLabel" name="labelDeathCause">
             <property name="text">
              <string>Причина:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDeathCause"/>
           </item>
          </layout>
         </item>
         <item row="13" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="lineEditMother">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Фамилия 
 после замужества: </string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEditFather">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="whatsThis">
              <string/>
             </property>
             <property name="accessibleName">
              <string/>
             </property>
             <property name="accessibleDescription">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Отец:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditPatronymic"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Фамилия:</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QTextEdit" name="textEditNote"/>
         </item>
         <item row="16" column="1">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButtonEditMother">
             <property name="text">
              <string>Ред.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSearchMother">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMotherDelete">
             <property name="text">
              <string>Х</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="11" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="pushButtonEditFather">
             <property name="text">
              <string>Ред.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSearchFather">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonFatherDelete">
             <property name="text">
              <string>Х</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="12" column="1">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Мать:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Супруги:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidgetSpouses"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="pushButtonAddSpouse">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Редактировать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDeleteMarriage">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Братья и сестры:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddSibling">
             <property name="text">
              <string>Добавить брата/сестру</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidgetSiblings"/>
         </item>
         <item>
          <widget class="Line" name="line_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Дети:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidgetChildren"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButtonChild">
             <property name="text">
              <string>Новый ребёнок</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonChildSearch">
             <property name="text">
              <string>Найти и добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutPlaces">
           <item>
            <widget class="QListWidget" name="listWidgetPlaces"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QPushButton" name="pushButtonPlaceAdd">
               <property name="text">
                <string>Добавить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonPlaceAddLast">
               <property name="text">
                <string>Последнее</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonPlaceEdit">
               <property name="text">
                <string>Редактировать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonPlaceDelete">
               <property name="text">
                <string>Удалить</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="text">
          <string>Удалить человека</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUnionHumans">
         <property name="text">
          <string>Объединить с</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSearchRecords">
         <property name="text">
          <string>Упоминания</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOk">
         <property name="text">
          <string>Добавить и закрыть</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditPatronymic</tabstop>
  <tabstop>lineEditSurName</tabstop>
  <tabstop>lineEditMarName</tabstop>
  <tabstop>comboBoxSex</tabstop>
  <tabstop>pushButtonSearchPlace</tabstop>
  <tabstop>checkBoxLife</tabstop>
  <tabstop>lineEditDeathDate</tabstop>
  <tabstop>lineEditDeathCause</tabstop>
  <tabstop>lineEditOccupation</tabstop>
  <tabstop>textEditNote</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>pushButtonAddSpouse</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButtonDeleteMarriage</tabstop>
  <tabstop>pushButtonChild</tabstop>
  <tabstop>pushButtonChildSearch</tabstop>
  <tabstop>listWidgetSpouses</tabstop>
  <tabstop>listWidgetChildren</tabstop>
  <tabstop>lineEditBirthdate</tabstop>
  <tabstop>lineEditBirthPlace</tabstop>
  <tabstop>lineEditFather</tabstop>
  <tabstop>lineEditMother</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
